namer/core/src/main/scala/io/buoyant/namer/ConstTransformer.scala

```diff
@@ -0,0 +1,14 @@
+package io.buoyant.namer
+
+import com.twitter.finagle.Name.Bound
+import com.twitter.finagle.{NameTree, Namer, Path}
+import com.twitter.util.Activity
+
+/** Bind the given path and use that instead of the tree. */
+class ConstTransformer(path: Path) extends NameTreeTransformer {
+
+  private[this] val bound = Namer.global.bind(NameTree.Leaf(path))
```

Comment:
This should have been NameInterpreter.global; I've updated.  We set NameInterpreter.global so this will be able to use configured namers.